extension/src/util/config.ts

```diff
@@ -28,6 +29,19 @@ export function getConfigGradleJavaHome(): string | null {
     return getConfigJavaImportGradleJavaHome() || getJdtlsConfigJavaHome() || getConfigJavaHome();
 }
 
+export function getJavaHome(): string | undefined {
+    return getConfigGradleJavaHome() || process.env.JAVA_HOME;
+}
+
+export async function checkEnvJavaExecutable(): Promise<boolean> {
```

Comment:
I don't see any async statements in the body.

Reply:
Let me remove "async" of this function and `getGradleServerEnv()`.